src/app/logic/processGame.js

```diff
@@ -0,0 +1,146 @@
+import Answers from '../components/Answers';
+import VisualImage from '../components/VisualImage';
+import { getQuestion, initGame, isGameInitialized } from './quiz';
+
+class Game {
```

Comment:
I like the idea od the use of class, we will store state in it.

Reply:
It doesn't need a constructor?

Reply:
Not really, initialization of fields and reset is done with newGame function